sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -491,6 +510,9 @@ case class CatalogTable(
     if (tableType == CatalogTableType.VIEW) {
       viewText.foreach(map.put("View Text", _))
       viewOriginalText.foreach(map.put("View Original Text", _))
+      if (SQLConf.get.viewSchemaBinding != "UNSUPPORTED") {
```

Comment:
Shall we read from the properties instead of getting it from the SQLConf?

Reply:
That would not allow us to switch the feature off for DESCRIBE.